src/components/Statements.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useParams } from 'react-router';
+
+const Statements = () => {
```

Comment:
- [ ] Your **Statements** page is missing the banner section as contained in the design, kindly add it.

  Your Design | Behance Design
  :---:|:---:
  ![vechin-own-detail](https://user-images.githubusercontent.com/36542986/152468740-027167ac-0752-4d6a-a14f-817684df9088.jpg) | ![vechin-orig-detail](https://user-images.githubusercontent.com/36542986/152468736-efbaa491-0128-4274-8acd-13379d2806bc.jpg)